assets/components/src/image-upload/index.test.js

```diff
@@ -6,16 +6,6 @@ import ImageUpload from './';
 describe( 'ImageUpload', () => {
 	it( 'should render an add image button', () => {
 		const { getByText } = render( <ImageUpload /> );
-		expect( getByText( 'Add image' ) ).toBeInTheDocument();
-	} );
-
-	it( 'should render an image and remove button', () => {
-		const image = {
-			id: 1234,
-			url: 'https://upload.wikimedia.org/wikipedia/en/a/a9/Example.jpg',
-		};
-		const { getByAltText, getByText } = render( <ImageUpload image={ image } /> );
-		expect( getByAltText( 'Upload preview' ) ).toBeInTheDocument();
-		expect( getByText( 'Remove image' ) ).toBeInTheDocument();
+		expect( getByText( 'Upload' ) ).toBeInTheDocument();
```

Comment:
Should we add a test for the remove button and background image when there's an image prop?

Reply:
👌  Added in 967f3b788d6558a421a5f182162719228d3b9dff